git-hub

```diff
@@ -179,22 +221,65 @@ def get_info(pr_num):
         labels = "None"
         if(pr_dictionary['labels'] is not ""):
             labels = pr_dictionary['labels']
-        click.echo('{0} \x1b[0;32;40m {1} \x1b[0m {2}/{3} {4}'.format(key, open_PR, pr_dictionary['user'], pr_dictionary['branch'], pr_dictionary['comment']))
-        click.echo('-Labels: {0}  -Reviewers: {1}  -Assignees: {2}  -Milestones: {3}'.format(labels, reviewers, assignee, milestone))
-
+        branch = "None"
+        if(pr_dictionary['branch'] is not ""):
```

Comment:
Or `branch = pr_dictionary.get('branch', 'None')`

Same for all the statements below.